client-gateway/bot/drive_ops_bot.py

```
@@ -0,0 +1,151 @@
+        "💬 **Крок 3/3**: Додайте коментар (під'їзд, дитяче крісло тощо) або натисніть кнопку нижче:", 
+        reply_markup=skip_menu(),
+        parse_mode='Markdown'
+    )
+    bot.register_next_step_handler(msg, process_comment_step)
+
+def process_comment_step(message):
+    chat_id = message.chat.id
+    
+    # Обробка опціонального поля
+    if message.text == "⏩ Пропустити":
+        user_orders[chat_id]['comment'] = "Не вказано"
```

Ruff (0.14.10)

String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)